app/uk/gov/hmrc/customs/declaration/model/actionbuilders/actionBuildersModel.scala

```diff
@@ -148,7 +148,7 @@ trait HasXmlBody {
   val xmlBody: NodeSeq
 }
 
-case class FileUploadRequest(declarationId: DeclarationId, fileGroupSize: FileGroupSize, files: Seq[FileUploadFile])
+case class FileUploadRequest(declarationId: DeclarationId, fileGroupSize: FileGroupSize, files: scala.collection.Seq[FileUploadFile])
```

Comment:
Can we leave as is? And use `scala.collection.immutable.Seq` throughout the rest of the code?

Reply:
I will give this a go